lib/claim/fees-claimant.js

```diff
@@ -55,13 +58,19 @@ class FeesClaimant {
      * @param {Currency} currency - The currency
      * @param {number} firstAccrual - The lower accrual index boundary
      * @param {number} lastAccrual - The upper accrual index boundary
-     * @returns {Promise} A promise that resolves into a BigNumber value representing the claimable amount of fees.
+     * @returns {Promise} A promise that resolves into a string value representing the claimable amount of fees.
      */
     async claimableFeesForAccruals(wallet, currency, firstAccrual, lastAccrual) {
         const ensemble = _tokenHolderRevenueFundContractEnsemble.get(this);
-        return ensemble.claimableAmountByAccruals(
+        const provider = _provider.get(this);
+
+        const claimableAmount = await ensemble.claimableAmountByAccruals(
             wallet, currency, firstAccrual, lastAccrual
         );
+
+        const tokenInfo = await provider.getTokenInfo(currency.ct.toString(), true);
+
+        return tokenInfo ? formatUnits(claimableAmount, tokenInfo.decimals) : claimableAmount.toString();
```

Comment:
At what situations do we expect `tokenInfo` to be undefined?
If `tokenInfo` is undefined, are we sure that it makes sense to just convert  `claimableAmount` to a string?
If not, would i make sense to throw an Error?

Reply:
There is a parallel case in https://github.com/hubiinetwork/nahmii-sdk/blob/master/lib/wallet/wallet.js#L82. Albeit in that case there is no conversion of BN amount to string. At least that case suggests a scenario in which there are monetary amounts (through balances) in nahmii that are not of supported token types.